src/applications/vaos/actions/appointments.js

```diff
@@ -101,7 +101,7 @@ export function cancelAppointment(appointment) {
   };
 }
 
-const BOOKED_REQUEST = 'Booked';
+const SUBMITTED_REQUEST = 'Submitted';
```

Comment:
I don't know what I was thinking here. You cancel Submitted requests, not booked ones.